File: Tienda_JAVA/src/javaapplication45/Electrodomesticos.java

package javaapplication45;

public class Electrodomesticos extends Producto {
    private double voltaje;
    private static int ID = 300;
    int id;
    public Electrodomesticos( String nombre, double precio, String fechaImportacion, int stock, double voltaje){
        super( nombre,  precio,  fechaImportacion,  stock);
        this.voltaje = voltaje;
        this.id=ID;
        ID++;
    }

    public int getId() {
        return id;
    }
    
    public String ObtenerInformaciopnconVoltaje(){
    return super.obtenerInformacion() +", voltaje "+ voltaje+ "ID: "+id;
}
}

File: Tienda_Pyton/Electrodomesticos.py
from Producto import Producto

class Electrodomesticos(Producto):
    ID = 300

    def __init__(self, nombre, precio, fecha_importacion, stock, voltaje):
        super().__init__(nombre, precio, fecha_importacion, stock)
        self.voltaje = voltaje
        self.id = Electrodomesticos.ID
        Electrodomesticos.ID += 1

    def get_id(self):
        return self.id

    def obtener_informacion_con_voltaje(self):
        return super().obtener_informacion() + ", voltaje " + str(self.voltaje) + " ID: " + str(self.id)
